.github/workflows/linux_riscv.yml

```diff
@@ -0,0 +1,188 @@
+          # generic dependencies
+          apt --assume-yes install cmake ccache ninja-build fdupes patchelf
+
+          python3 -m venv ${OPENVINO_BUILD_DIR}/env
+          source ${OPENVINO_BUILD_DIR}/env/bin/activate
+          python3 -m pip install -r ${OPENVINO_REPO}/src/bindings/python/requirements.txt
+          python3 -m pip install -r ${OPENVINO_REPO}/src/bindings/python/wheel/requirements-dev.txt
+          python3 -m pip install -r ${OPENVINO_REPO}/src/bindings/python/src/compatibility/openvino/requirements-dev.txt
+          python3 -m pip install conan
+
+      - name: Install RISC-V native debian packages
+        run: |
+          echo deb [arch=amd64] http://archive.ubuntu.com/ubuntu/ jammy main restricted > riscv64-sources.list
+          echo deb [arch=amd64] http://archive.ubuntu.com/ubuntu/ jammy-updates main restricted >> riscv64-sources.list
+          echo deb [arch=amd64] http://archive.ubuntu.com/ubuntu/ jammy universe >> riscv64-sources.list
+          echo deb [arch=amd64] http://archive.ubuntu.com/ubuntu/ jammy-updates universe >> riscv64-sources.list
+          echo deb [arch=amd64] http://archive.ubuntu.com/ubuntu/ jammy multiverse >> riscv64-sources.list
+          echo deb [arch=amd64] http://archive.ubuntu.com/ubuntu/ jammy-updates multiverse >> riscv64-sources.list
+          echo deb [arch=amd64] http://archive.ubuntu.com/ubuntu/ jammy-backports main restricted universe multiverse >> riscv64-sources.list
+          echo deb [arch=amd64] http://security.ubuntu.com/ubuntu/ jammy-security main restricted >> riscv64-sources.list
+          echo deb [arch=amd64] http://security.ubuntu.com/ubuntu/ jammy-security universe >> riscv64-sources.list
+          echo deb [arch=amd64] http://security.ubuntu.com/ubuntu/ jammy-security multiverse >> riscv64-sources.list
+          echo deb [arch=riscv64] http://ports.ubuntu.com/ubuntu-ports/ jammy main >> riscv64-sources.list
+          echo deb [arch=riscv64] http://ports.ubuntu.com/ubuntu-ports/ jammy universe >> riscv64-sources.list
+          echo deb [arch=riscv64] http://ports.ubuntu.com/ubuntu-ports/ jammy-updates main >> riscv64-sources.list
+          echo deb [arch=riscv64] http://ports.ubuntu.com/ubuntu-ports/ jammy-security main >> riscv64-sources.list
+          mv riscv64-sources.list /etc/apt/sources.list.d/
+          dpkg --add-architecture riscv64
+          apt-get update -o Dir::Etc::sourcelist=/etc/apt/sources.list.d/riscv64-sources.list
+          apt-get install -y --no-install-recommends libpython3-dev:riscv64
+
+      - name: Create conan_toolchain.cmake file
+        run: |
+          source ${OPENVINO_BUILD_DIR}/env/bin/activate
+          # generate build profile
+          conan profile detect
+          # patch settings.yml to contain riscv64 architecture
+          sed -i 's/sparcv9/riscv64/g' ~/.conan2/settings.yml
+          # generate host profile for linux_riscv64
+          echo "include(default)" > ${OPENVINO_BUILD_DIR}/linux_riscv64
```

Comment:
extract `${OPENVINO_BUILD_DIR}/linux_riscv64` in an env var, just for this step? It is used quite a lot; might prevent misspellings.

Reply:
ok, will update in next PR

Reply:
fixed here https://github.com/openvinotoolkit/openvino/pull/20078